src/components/tutorials-landing-view/types.ts

```diff
@@ -0,0 +1,42 @@
+import { ProductSlug } from 'types/products'
+import { type BadgeProps } from 'components/badge'
+
+interface ContentCardLinkBadge {
+	icon: BadgeProps['icon']
+	label: BadgeProps['ariaLabel']
+}
+
+type ContentCardLinkEyebrowPart = string
```

Comment:
Reviving my question so it can be part of my review:

> 💭 What's the point of this type alias? Not sure I understand the value of that here.

> `ContentCardLinkEyebrowPart` is used alongside `eyebrowParts.map()` in the markup... So if the type of `eyebrowParts` changes, from a `string` to an object for example, then there's one less thing in the markup to update.
> https://github.com/hashicorp/dev-portal/pull/1957#discussion_r1206942333

⛏️ Warning - nitpick ahead:

I largely don't have strong opinions about _this_ occurrence, because it is small and I am confident we won't run into any issues. If we extrapolate this to _every time a primitive type is referenced in MDX, we should defensively alias it in the event it changes in the future_, I would disagree with this approach.

Largely, when a type changes, I **want** my editor telling me that the type is incorrect in usage (e.g a map) because I likely have other things to address around it.

Following this concrete example, in the event we re-alias your `ContentCardLinkEyebrowPart` to a more complex object, the following code below won't highlight anything as an issue, and it would happily render your complex object `{eyebrowPart}` in object form.

https://github.com/hashicorp/dev-portal/pull/1957/files#diff-a4bde531afbc58b87b80cb9dc416ebaaef1c951f3a039e6f037cb70cfd569d26R70-R79

Reply:
So when I change the type of `ContentCardLinkEyebrowPart` to an object, here's the TS error I see:

![image](https://github.com/hashicorp/dev-portal/assets/43934258/1acca42a-0f6d-4f6b-b55b-3eb24bdaed38)

> If we extrapolate this to _every time a primitive type is referenced in MDX, we should defensively alias it in the event it changes in the future_, I would disagree with this approach.

I'd probably disagree with a blanket statement as well, but this isn't meant to be a global proposal! Just felt comfortable in this case. 😊

Reply:
Cool cool thanks for hearing me out 😄 ; no need to change this here!